testing/codeflare/configure.sh

```diff
@@ -13,7 +13,7 @@ fi
 
 TESTING_UTILS_DIR="$TESTING_THIS_DIR/../utils"
 
-export CI_ARTIFACTS_FROM_COMMAND_ARGS_FILE=${TESTING_THIS_DIR}/command_args.yaml
+export CI_ARTIFACTS_FROM_COMMAND_ARGS_FILE=${TESTING_THIS_DIR}/command_args.yml.j2
```

Comment:
good catch, @dagrayvid noticed that the extension was wrong not so long ago. My emacs don't mind the Jinja markup :D